### ISSUE TITLE
Cannot fork() a new process: Resource temporarily unavailable (errno=11)

### ISSUE THREAD

**Issue by MatsG23** [phusion/passenger#2354]
## Issue report

**Question 1: What is the problem?**
Expected behaviour: My application should run continuously

Actual behaviour: The web application sometimes can't be started. I can only fix it by restarting nginx. It feels like the problem don't follow a pattern. Error logs I found:
```
Warning: unable to write to /tmp/passenger.spawn.XXXXdPIFe7/response/steps/subprocess_wrapper_preparation/begin_time: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (1618479049.104)
Warning: unable to write to /tmp/passenger.spawn.XXXXdPIFe7/response/steps/subprocess_wrapper_preparation/end_time: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (1618479049.104)
Warning: unable to write to /tmp/passenger.spawn.XXXXdPIFe7/response/steps/subprocess_app_load_or_exec/begin_time: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (1618479049.105)
[ E 2021-04-15 11:30:49.2014 5197/To age/Cor/App/Implementation.cpp:221 ]: Could not spawn process for application /var/www/vhosts/...: Cannot fork() a new process: Resource temporarily unavailable (errno=11)
```
**Question 2: Passenger version and integration mode**:
open source 6.0.6/nginx

**Question 3: OS or Linux distro, platform (including version)**:
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.5 LTS
Release:          18.04
Codename:     bionic

**Question 4: Passenger installation method**:
[ X] Other, please specify: In Plesk UI (server managment system)

**Question 5: Your app's programming language (including any version managers) and framework (including versions)**:
Node.js 15.9.0 + Express.js 4.17.1

**Question 6: Are you using a PaaS and/or containerization? If so which one?**
No it's a simple vServer

**Question 7: Anything else about your setup that we should know?**
No

I hope I did it right, I didn't write issues that often

**Comment by CamJN** [comment]
The warnings are fixed in 6.0.7, the error is because your system cannot fork, that's something you'll have to investigate further.

**Comment by MatsG23** [comment]
It sadly seems like I couldn't manually upgrade Passenger on my Ubuntu server with Plesk. Last time I tried, it made much problems.
The "Can't fork" stuff comes across me since few days. The server has enough RAM and storage left, but these limits on linux block some of the processes' work. 
Do you have any suggestions? My plan is to turn all limits off or set them to unlimited... I don't understand why there are limits like that
